stanford-earth/SE3_BLT
# Earth 408 shea

# READY FOR REVIEW

# Summary
- Merged several of the PRs in to one as they were related
- Merged the two htaccess patches in to one file
- Added new BLT command to run just the saml symlink command
- Changed HTTPs redirect logic (up for discussion)
- Adds custom.settings.php file and patch for settings.php for memcache settings
- Adds call to drupal-scaffold on every `composer install / composer update` to ensure it gets run

# Needed By (Date)
- Sometime before too long.

# Steps to Test

1. Check out branch
2. Trash your docroot
3. Run `composer install --prefer-source`
4. Add your DB settings to settings.php
5. Validate .htacess file, settings.php, and simplesamlphp symlink

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
